README.md

```diff
@@ -57,7 +64,7 @@ HTTP/1.1 202 Accepted
 {"favorite_dog_breed": "dogfish"}
 ```
 
-###Using with NSURLSessionConfiguration
+### Using with NSURLSessionConfiguration
 
 In order to get `NSURLSesssion` to use `VOKMockUrlProtocol`, you must insert it's class into a `NSURLSessionConfigurations`'s [`protocolClasses`](https://developer.apple.com/library/prerelease/ios/documentation/Foundation/Reference/NSURLSessionConfiguration_class/index.html#//apple_ref/occ/instp/NSURLSessionConfiguration/protocolClasses).
```

Comment:
Ugh.  Whoever reviewed that line when it was added should have been more careful.  (That couldn't possibly have been my "LGTM" on that pull request...  I reject your reality and substitute my own.)